### PR TITLE
Add recipe for gistim package

### PR DESCRIPTION
**Pull request by Huite**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] ~If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/gistim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  summary: GIS utilities for Tim(ML) Analytic Element modeling
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+  doc_url: https://deltares.gitlab.io/imod/qgis-tim
+  dev_url: https://gitlab.com/deltares/imod/qgis-tim
+
+extra:
+  recipe-maintainers:
+    - Huite
+    - visr
```
Comment (marcelotrevisani) [inline recipes/gistim/meta.yaml:56]: Hi @visr , could you please confirm that you would like to maintain this recipe as well?

Comment (visr) [reply recipes/gistim/meta.yaml:56]: I confirm that I'd like to maintain this recipe.

**Comment by visr** [comment]
@conda-forge/staged-recipes could you please have another look? The review comment and linting issues have been addressed. Thanks!

File: recipes/gistim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - affine
+    - geopandas
+    - matplotlib
```
Comment (chrisburr) [inline recipes/gistim/meta.yaml:25]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib
```suggestion
    - matplotlib-base
```

**Pull request merged**